components/Ranking/index.js

```diff
@@ -0,0 +1,139 @@
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+
+export default function Ranking({ position }) {
+  const router = useRouter();
+  const { isReady } = router;
+  const {
+    data: players,
+    isLoading,
+    error,
+  } = useSWR("/api/players", {
+    fallbackData: [],
+  });
+
+  if (!isReady || isLoading || error)
```

Comment:
The `isReady` property of the `router` object should be used in `useEffect`.

I'd say, if you experience errors, you might want to add one such `useEffect` and check the value of `isReady` within the callback. If you're not seeing any errors, you may leave the code as is.